keepalived-vip/keepalived.tmpl

```diff
@@ -27,8 +32,17 @@ vrrp_instance vips {
   virtual_ipaddress { {{ range .vips }}
     {{ . }}{{ end }}
   }
+
+{{ if .proxyMode}}
+  # In proxy mode there is no need to create virtual servers
+  track_script {
+    chk_haproxy weight 1
+  }
+{{ end }}
+
 }
 
+{{ if not .proxyMode }}
```

Comment:
This means that **every** service listed in the configmap will use a forwarding type of PROXY, even if "DR" is explicitly stated on any other service. Is this the intended behaviour?